README.md

```diff
@@ -295,9 +295,9 @@ Commit, tag and create a new release:
 
 ```sh
 git add CHANGELOG.md README.md
-git commit -m "Bump v4.4.4"
+git commit -m "Bump v4.4.6"
```

Comment:
You should replace all occurence of 4.4.4 to 4.4.5 in this file. There are more than these two, moreover you did a mistake in the version here. Please use the replace all feature of your IDE to replace all occurences

Reply:
I indeed missed the top of the file that was still in v4.4.4
However, shouldn't the "bump to xxx" code block use the next version instead of the current one ?